Python/crossfire/README.md

```diff
@@ -72,3 +72,16 @@ For more information on how the package works and for exemples on using the modu
 ## Contributors
 
 [Sérgio Spagnuolo](https://github.com/voltdatalab), [Denisson Silva](https://github.com/silvadenisson) and [Felipe Sodré Mendes Barros](https://github.com/FelipeSBarros).
+
+# Contributing
+
+## Code Formatting
+
+To ensure everyone uses same code style, we suggest applying [Black](https://black.readthedocs.io/en/stable/index.html) before sending your code enhancement.
+
+### Installing project's development dependencies:
+
+```commandline
+poetry install --with dev
+black .
+```
```

Comment:
The `black .` command does not match the title (install dev dependencies)

What about?


> To ensure everyone uses the same code style, we suggest applying [Black](https://black.readthedocs.io/) before sending your code enhancement. After installing the dependencies with [Poetry](https://python-poetry.org/):
> 
>```console
>$ poetry run black . --verify
>```

Reply:
> ```
> poetry run black . --verify
> ```

Tentei rodar aqui e acusou não existir o `--verify`; Olhando o --help, vi que existe o `--check`.
Seria ele?
```
 poetry run black . --check
All done! ✨ 🍰 ✨
6 files would be left unchanged.
```

Reply:
Coloquei como --check, usando o git reset, OK?

Reply:
As alterações, na minha opinião, mantém o problema que apontei:

> The `black .` command does not match the title (install dev dependencies)

Pq temos um título _Installing project's development dependencies_ se os comandos fazem **mais** do que instalar as dependências?

Pq nas instruções para verificar o estilo, temos que instalar as dependências? Vamos precisar instalar toda vez que formos verificar o estilo?

Reply:
Perfeito. Agora entendi o seu ponto.
O commi anterior eu fui arrojado e fiz um revevrt para editar o commit. Com isso tive que forçar o push.
Entendo que não é uma boa prática. Fiz essa última alteração em um novo commit.
Acho que estamos